REST.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "REST (Representational State Transfer)"
+date: 2020-10-28
+part-of-speech: noun
+synonyms: ["RESTful"]
+abbreviation: REST
+---
+
+Representational state transfer (REST) is a software architectural style that defines a set of constraints to be used for creating Web services. Web services that conform to the REST architectural style, called RESTful Web services, provide interoperability between computer systems on the internet. 
```

Comment:
Thanks for your contribution! This paragraph is a word-for-word copy from [Wikipedia](https://en.wikipedia.org/wiki/Representational_state_transfer).  Do you mind writing it in your own words? We'd like to avoid directly copying others' words. Thanks for your revision!

Reply:
> Thanks for your contribution! This paragraph is a word-for-word copy from [Wikipedia](https://en.wikipedia.org/wiki/Representational_state_transfer). Do you mind writing it in your own words? We'd like to avoid directly copying others' words. Thanks for your revision!

Sure, I will write it in my own words. I was not aware about avoiding copying from Wikipedia. Thanks for your review.